reana/cli.py

```diff
@@ -1047,3 +1059,40 @@ def run_example(component, workflow_engine, output, sleep):  # noqa: D301
                 run_command(cmd, component)
     # report status; everything was OK
     run_command('echo OK', component)
+
+
+@click.option('--component', '-c', multiple=True, default=['ALL'],
+              help='Which components? [shortname|name|.|CLUSTER|ALL]')
+@cli.command(name='kubectl-delete-pod')
+def kubectl_delete_pod(component):  # noqa: D301
+    """Delete REANA component's pod.
+
+    If option ``component`` is not used, all pods will be deleted.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-job-controller';
+                         * (2) short component name such as 'r-j-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
+                         * (5) special value 'CLIENT' that will expand to
+                               cover all REANA client components;
+                         * (6) special value 'ALL' that will expand to include
+                               all REANA repositories.
+    :type component: str
+
+    """
+    if "ALL" in component:
+        cmd = 'kubectl delete --all pods'
+        run_command(cmd)
+    else:
+        components = select_components(component)
+        for component in components:
+            if component in COMPONENT_PODS:
+                cmd = 'kubectl delete pod -l app={0}'.format(
```

Comment:
Please add `--wait=false` to the command options for faster action.

Otherwise this command waits for pod deletion. Keeping our developer-oriented use case, I think we may want to return as soon as possible.  This would permit to plug the command into your editing pipeline, e.g. edit some file in `r-j-c`, save it, which would automatically trigger pod deletion via this command... and we wouldn't want our editors to wait. We could use asynchronous OS process for this, but adding `--wait=false` seems just as good and simpler.

```console
$ time kubectl delete pod -l app=job-controller              
                                                                                  
pod "job-controller-5fcb4c5c9c-djscz" deleted
kubectl delete pod -l app=job-controller  0.09s user 0.01s system 0% cpu 32.204 total
$ time kubectl delete pod --wait=false -l app=job-controller  
                                                               
pod "job-controller-5fcb4c5c9c-fhhqk" deleted
kubectl delete pod --wait=false -l app=job-controller  0.15s user 0.03s system 111% cpu 0.159 total
```